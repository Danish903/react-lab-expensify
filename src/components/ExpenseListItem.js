import React from 'react';

const Item = ({ description, amount, createdAt }) => (
  <div>
    <h4>{description}</h4>
    <p>
      ${amount} - {createdAt}
    </p>
  </div>
);

export default Item;
