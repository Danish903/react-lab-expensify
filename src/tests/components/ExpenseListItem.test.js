import React from 'react';
import { shallow } from 'enzyme';
import ExpenseListItem from '../../components/ExpenseListItem';
import expenses from '../fixures/expenses';

test('should render ExpenseListItem with expense', () => {
  const wrapper = shallow(
    <ExpenseListItem {...expenses[1]} key={expenses[1].id} />
  );
  expect(wrapper).toMatchSnapshot();
});
