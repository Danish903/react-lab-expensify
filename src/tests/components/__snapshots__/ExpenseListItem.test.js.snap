// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseListItem with expense 1`] = `
<Link
  className="list-item"
  replace={false}
  to="/edit/2"
>
  <div>
    <h3
      className="list-item__title"
    >
      April Rent
    </h3>
    <span
      className="list-item__subtitle"
    >
      December 27th, 1969
    </span>
  </div>
  <h3
    className="list-item__data"
  >
    $1,020.00
  </h3>
</Link>
`;
